src/Components/AddMaterialPage/AddMaterialPage.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
+import styles from './AddMaterialPage.css';
+
+const AddMaterialPage = ({ onAddMaterial }) => {
+  const [fileLink, setFileLink] = useState('');
+  const [externalLink, setExternalLink] = useState('');
+  const [file, setFile] = useState(null);
+  const [compendium, setCompendium] = useState('');
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    onAddMaterial({ fileLink, externalLink, file, compendium });
+    setFileLink('');
+    setExternalLink('');
+    setFile(null);
+    setCompendium('');
+  };
+
+  const handleFileChange = (e) => {
+    setFile(e.target.files[0]);
+  };
+
+  return (
+    <div className={styles.addMaterialPage}>
+      <h1>New Information</h1>
```

Comment:
Do we need it to be `h1`? I would suggest replacing it with a level 2 header (`h2`). It would help to avoid two h1 on the same page.